src/Header/CdrEofHeader.php

```diff
@@ -0,0 +1,71 @@
+<?php
+declare(strict_types = 1);
+
+namespace ZipStream\Header;
+
+/**
+ * Class CdrEofHeader
+ * @package ZipStream\Header
+ */
+class CdrEofHeader
+{
+    use HeaderHelperTrait;
+
+    /**
+     * @var int
+     */
+    private $offset;
+
+    /**
+     * @var int
+     */
+    private $headerOffset;
+
+    /**
+     * @var int
+     */
+    private $fileCount;
+
+    /**
+     * CdrEofHeader constructor.
+     * @param int $offset
+     * @param int $headerOffset
+     * @param int $fileCount
+     */
+    public function __construct($offset, $headerOffset, $fileCount)
+    {
+        $this->offset = $offset;
+        $this->headerOffset = $headerOffset;
+        $this->fileCount = $fileCount;
+    }
+
+    /**
+     * Send CDR EOF (Central Directory Record End-of-File) record.
+     *
+     * @param resource $outputStream
+     * @param string   $comment
+     */
+    public function write($outputStream, string $comment)
+    {
+        $data = $this->packFields($this->getFields($comment)) . $comment;
+        fwrite($outputStream, $data);
+    }
+
+    /**
+     * @param string $comment
+     * @return array
+     */
+    private function getFields(string $comment): array
```

Comment:
I think $comment should be passed in the constructor, prob it's not more/less special than fileCount, offset,etc, and either there should be one comment or none. Same for write() method. Thoughts?